### PR TITLE
Adds support for displaying multiple phone numbers by contact type. 

### PR DESCRIPTION
**Pull request by tomthorogood**
(Does not yet add support for phone searches, that's next.)

### CODE REVIEW

File: husky_directory/models/search.py [hunk 1]
```diff
@@ -28,19 +27,24 @@ class SearchDirectoryInput(DirectoryBaseModel):
     phone: Optional[str] = Field(None, regex=r"^[0-9]+$", max_length=32)
```
Comment (goulter) [inline husky_directory/models/search.py:27]: Line 27 and 43, what the difference between phone and phone contacts?
Also, is phone optional but phone contacts is not? Sorry just need some clarification.

Comment (tomthorogood) [reply husky_directory/models/search.py]: `phone` should have been removed.

Comment (tomthorogood) [reply husky_directory/models/search.py]: I believe you review this again you'll find this is already taken care of. I had a bit of a rebasing issue.

File: husky_directory/models/search.py [hunk 2]
```diff
@@ -29,19 +28,24 @@ class SearchDirectoryInput(DirectoryBaseModel):
...
+    voice_mails: List[str] = Field([], alias="voicemail")
+    touch_dials: List[str] = Field([], alias="tdd")
+    mobiles: List[str] = Field([], alias="mobile")
+    pagers: List[str] = Field([], alias="pager")
+
+
 class Person(DirectoryBaseModel):
     name: str
-    phone: Optional[str]
+    phone_contacts: PhoneContactMethods = PhoneContactMethods()
```
Comment (goulter) [inline husky_directory/models/search.py:44]: line 44
Great, only one phone field now. `phone` was type `Optional` before and with the new type `PhoneContactMethods `, without the `Optional` type is this field now required?

Comment (tomthorogood) [reply husky_directory/models/search.py:44]: No, a default is provided. If you look up `Optional` you'll find that it's just an alias that also accepts None.

Comment (tomthorogood) [reply husky_directory/models/search.py:44]: And remember: this is _output_, not input. You won't be building these objects.

Comment (tomthorogood) [reply husky_directory/models/search.py:44]: https://pydantic-docs.helpmanual.io/usage/types/#standard-library-types

Comment (tomthorogood) [reply husky_directory/models/search.py:44]: I highly recommend reading and familiarizing yourself with the pydantic docs. They're quite great. When I started learning pydantic I read them "cover to cover".

**Review by goulter — APPROVED** [review]

File: husky_directory/templates/full_results.html [hunk 3]
```diff
@@ -14,7 +14,14 @@ <h4>{{scenario['description']}}</h4>
                     <h3>{{ data['name'] }}</h3>
                     <ul class="dir-listing">
                         <li>{{ data['email'] }}</li>
-                        <li>{{ data['phone'] }}</li>
```
Comment (goulter) [inline husky_directory/templates/full_results.html:17]: RE: change is displaying the phone(s)
This same kind of change is needed `summar_results.html`

**Pull request merged**